evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -185,6 +188,59 @@ def test_make_pipeline(
             ], test_description
 
 
+@pytest.mark.parametrize(
+    "problem_type",
+    [
+        ProblemTypes.TIME_SERIES_BINARY,
+        ProblemTypes.TIME_SERIES_MULTICLASS,
+        ProblemTypes.TIME_SERIES_REGRESSION,
+    ],
+)
+@pytest.mark.parametrize("frequency", ["D", "MS", "A", "T"])
+def test_make_pipeline_controls_decomposer(
+    problem_type,
+    frequency,
+    get_test_data_from_configuration,
+):
+    X, y = get_test_data_from_configuration(
+        "ww",
+        problem_type,
+        column_names=["dates", "numerical"],
+    )
+    X.ww["dates"] = pd.Series(pd.date_range("2000-02-03", periods=20, freq=frequency))
+    parameters = {
+        "pipeline": {
+            "time_index": "date",
+            "gap": 1,
+            "max_delay": 1,
+            "forecast_horizon": 3,
+        },
+    }
+    estimators = get_estimators(problem_type=problem_type)
+    pipeline_class = _get_pipeline_base_class(problem_type)
+    for estimator_class in estimators:
+        if problem_type in estimator_class.supported_problem_types:
+
+            pipeline = make_pipeline(X, y, estimator_class, problem_type, parameters)
+            assert isinstance(pipeline, pipeline_class)
+
+            if is_regression(problem_type) and frequency in ["D", "MS"]:
```

Comment:
I thought we were catching "A" and "MS"?  If that's the case, I would think we might want to capture the list of frequencies as a module level list in the STLDecomposer (imported) and reference it here?  We can file that as a separate issue though.

Reply:
Annual and ten minute intervals are the frequencies we've found to fail at this point. MS seems to fail with the polynomial decomposer with the millisecond issue but does fine with STL. I do have the failing frequencies in a constant within utils, I'll reference that here!